support-frontend/assets/components/countryGroupSwitcher/countryGroupSwitcherContainer.jsx

```diff
@@ -28,16 +27,12 @@ export default function (
     })();
   }
 
-  function mapStateToProps(state: { common: CommonState }): PropTypes {
-
-    return {
-      countryGroupIds: listOfCountries,
-      selectedCountryGroup: state.common.internationalisation.countryGroupId,
-      subPath,
-      onCountryGroupSelect,
-    };
-  }
-
-  return connect(mapStateToProps)(CountryGroupSwitcher);
-
+  return () => (
```

Comment:
I wonder do we need the `CountryGroupSwitcherContainer` anymore? Could we replace instances where we use it by using `CountryGroupSwitcher` directly instead?